srtcore/api.cpp

```diff
@@ -782,19 +906,45 @@ SRTSOCKET CUDTUnited::accept(const SRTSOCKET listen, sockaddr* pw_addr, int* pw_
    return u;
 }
 
-int CUDTUnited::connect(const SRTSOCKET u, const sockaddr* name, int namelen, int32_t forced_isn)
+int CUDTUnited::connect(SRTSOCKET u, const sockaddr* srcname, int srclen, const sockaddr* tarname, int tarlen)
```

Comment:
This is an extra version of `connect` that maps to `srt_connect_bind`. The latter is used as a combination of `bind` and `connect` (simple replacement in case of sockets). However this bind-then-connect cannot be used this way for the socket groups, therefore it needs an extra function for it.